Controller/Blog.php

```diff
@@ -26,6 +26,7 @@ public function __construct()
         /** Get the Model class in all the controller class **/
         $this->oUtil->getModel('Blog');
```

Comment:
I'm pretty sure you will need `$this->oUtil->getModel('Comment');` after `$this->oUtil->getModel('Blog');` in order to include the file, since the SPL autoload isn't done for the Model classes.